CorvallisBus.Core/WebClients/GoogleTransitClient.cs

```diff
@@ -129,7 +129,7 @@ private static List<GoogleRouteSchedule> ParseScheduleCSV(ZipArchiveEntry entry)
                     {
                         route = s_routePattern.Match(line[0]).Groups[1].Value,
                         stop = line[3],
-                        order = line[4],
+                        order = int.Parse(line[4]),
```

Comment:
int.Parse is not exactly the robust solution here but the job might as well fail if something besides a number is used as an ordering value in the CSV.